src/main/java/org/antipathy/mvn_scalafmt/FormatMojo.java

```diff
@@ -25,18 +28,22 @@
     private boolean skipSources;
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
+    @Parameter(defaultValue = "${project.build.sourceDirectory}", required = true)
+    private List<File> sourceDirectories;
+    @Parameter(defaultValue = "${project.build.testOutputDirectory}", required = true)
+    private List<File> testSourceDirectories;
 
     public void execute() throws MojoExecutionException {
 
         ArrayList<Object> sources = new ArrayList<>();
         ArrayList<Object> testSources = new ArrayList<>();
 
         if (!skipSources) {
-            sources.addAll(project.getCompileSourceRoots());
+            sources.addAll(sourceDirectories);
         }
 
         if (!skipTestSources) {
-            testSources.addAll(project.getTestCompileSourceRoots());
+            testSources.addAll(testSourceDirectories);
         }
 
```

Comment:
Nice work, could you bump the version number in the pom and update the example in the readme to include these options?